packages/frontend/src/contexts/utils.js

```diff
@@ -30,3 +30,22 @@ export const request = async (network, query) => {
   }).then((res) => res.json())
   return res
 }
+
+export const shiftAttestations = async (
```

Comment:
If config is not loaded here, it does not have to be `async` function

Reply:
oh you're right, I just copied and pasted. I will remove it.